health/micro-ui/web/micro-ui-internals/packages/modules/health-dss/src/pages/employee/L2Main.js

```
@@ -0,0 +1,754 @@
+  // Return the id of the matching object or null if not found
+  const projectTypeId = matchingProject ? matchingProject.id : null;
+  return projectTypeId;
+}
+
+function getProjectTypeFromSession() {
+  const projectTypeSession = Digit.SessionStorage.get("projectSelected")?.project?.projectType;
+  // Return the id of the matching object or null if not found
+  const projectTypeCode = projectTypeSession ? projectTypeSession : null;
+  return projectTypeCode;
+}
+const L2Main = ({}) => {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)